_projects/roberta_external_source.md

```diff
@@ -60,21 +63,31 @@ additional certification through Fraunhofer IAIS in Sankt Augustin, Germany.
 
 ##### PREPARATION/BONDING
 
-* Familiarity with the OpenRoberta partner project is established and a method for external access is designed and implemented in a basic fashion
-* A design goal is reached with the coala project on how to integrate external data sources into the coala web page, including mock-ups and a prototype implementation for the UI
+* Familiarity with the OpenRoberta partner project is established and a method
+  for external access is designed and implemented in a basic fashion
+* A design goal is reached with the coala project on how to integrate external data sources into the coala web page,
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.